frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java

```diff
@@ -181,17 +179,9 @@ public void queueRepositoryUpdateAndGraphTableRefresh() {
     LOG.debug("Entering");
 
     if (project != null && !project.isDisposed()) {
-      // GitUtil.getRepositories(project) should never return empty list because it means there is no git repository
-      // in an opened project, so Git Machete plugin shouldn't even be loaded in the first place
-      @SuppressWarnings("value:assignment.type.incompatible")
-      @MinLen(1)
-      // A bit of a shortcut: we're accessing git repositories even though we may be on UI thread here;
```

Comment:
Yeah, this comment should be moved to the new location alongside with the `GitUtil.getRepositories` invocation